setup.cfg

```diff
@@ -34,6 +34,7 @@ packages =
     ansible_deployer
     ansible_deployer.modules
     ansible_deployer.modules.configs
+    ansible_deployer.modules.misc
```

Comment:
Am I correct that this commit shouldn't introduce any functionali change? That's something that is usually good to emphasize in commit message.